packages/overture-schema-places-theme/src/overture/schema/places/place.py

```diff
@@ -82,6 +82,50 @@ class Categories(BaseModel):
     ] = None
 
 
+@no_extra_fields
+class Taxonomy(BaseModel):
+    """
+    A structured representation of the place's category within the Overture taxonomy.
+
+    Provides the primary classification, full hierarchy path, and alternate categories.
+    """
+
+    # Required
+
+    primary: Annotated[
+        SnakeCaseString,
+        Field(
+            description="The most specific category for this place within the taxonomy hierarchy."
+        ),
+    ]
+
+    hierarchy: Annotated[
+        list[SnakeCaseString],
+        Field(
+            min_length=1,
+            description=textwrap.dedent("""
+                The full taxonomy path from the most general to most specific category,
+                represented as an ordered array (e.g., [food_and_drink, restaurant,
+                casual_eatery, gas_station_sushi]).
```

Comment:
*Uber* nits that we're certainly not being fully consistent about anyway, but hey, it's what I do...

- s/array/list in the docstring
- I'd be tempted to put backticks around the brackets and double quotes around the strings so you get

```
`["food_and_drink", "restaurant", ...]`
```

---

How I would p~~y~~edantically write the docs:

```
The c
```